include/sensors/core/tasks/humidity_task.hpp

```diff
@@ -0,0 +1,165 @@
+#pragma once
+
+#include "can/core/can_writer_task.hpp"
+#include "can/core/ids.hpp"
+#include "can/core/messages.hpp"
+#include "common/core/bit_utils.hpp"
+#include "common/core/logging.hpp"
+#include "common/core/message_queue.hpp"
+#include "pipettes/core/messages.hpp"
+#include "sensors/core/humidity_sensor.hpp"
+#include "sensors/core/utils.hpp"
+
+namespace humidity_task {
+
+using namespace humidity_utils;
+using namespace can_ids;
+
+template <message_writer_task::TaskClient CanClient>
+struct HumidityReadingCallback {
+    CanClient &can_client;
+
+    void operator()(const pipette_messages::MaxMessageBuffer &buffer) {
+        uint16_t data = 0x0;
+        const auto *iter = buffer.cbegin();
+        iter = bit_utils::bytes_to_int(iter, buffer.cend(), data);
+        auto humidity = humidity_utils::convert(data, SensorType::humidity);
+
+        auto message = can_messages::ReadFromSensorResponse{
+            {}, SensorType::humidity, static_cast<uint16_t>(humidity)};
+        can_client.send_can_message(can_ids::NodeId::host, message);
+    }
+};
+
+template <message_writer_task::TaskClient CanClient>
+struct TemperatureReadingCallback {
```

Comment:
building on @amitlissack 's comment above - can we just leave this unimplemented rather than making it the temperature sensor? kind of confusing